plugin/trino-openlineage/src/main/java/io/trino/plugin/openlineage/transport/kafka/OpenLineageKafkaTransport.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.openlineage.transport.kafka;
+
+import com.google.inject.Inject;
+import io.openlineage.client.transports.KafkaConfig;
+import io.openlineage.client.transports.KafkaTransport;
+import io.openlineage.client.transports.Transport;
+import io.trino.plugin.openlineage.config.kafka.OpenLineageClientKafkaTransportConfig;
+import io.trino.plugin.openlineage.transport.OpenLineageTransport;
+
+import java.util.Properties;
+
+public class OpenLineageKafkaTransport
+        implements OpenLineageTransport
+{
+    private final String topicName;
+    private final String messageKey;
+    private final String brokerEndpoints;
+    private final String bootstrapServers = "bootstrap.servers";
+    private final String acks = "acks";
+    private final String acksDefault = "all";
+    private final String keySerializer = "key.serializer";
+    private final String keySerializerDefault = "org.apache.kafka.common.serialization.StringSerializer";
+    private final String valueSerializer = "value.serializer";
+    private final String valueSerializerDefault = "org.apache.kafka.common.serialization.StringSerializer";
```

Comment:
How about having them as a static final variables ?

Reply:
For the Config key we could use something like 
```
        kafkaProperties.put(BOOTSTRAP_SERVERS_CONFIG, brokerEndpoints);
        kafkaProperties.put(ACKS_CONFIG, acksDefault);
        kafkaProperties.put(KEY_SERIALIZER_CLASS_CONFIG, keySerializerDefault);
        kafkaProperties.put(VALUE_SERIALIZER_CLASS_CONFIG, valueSerializerDefault);`

Reply:
In case of custom protocol - We might need to capture them as a seperate config

Reply:
Like `KafkaSslConfig` in Trino

Reply:
yeah but i would really avoid keeping defaults hardcoded without a way for a user to change, KafkaTransportConfig would be more flexible